src/pkg/gc/gc_test.go

```diff
@@ -29,9 +30,37 @@ func (d *FakeDiskSpaceFetcher) GetUsedDiskSpaceInPercents() (int, error) {
 	return d.counter, nil
 }
 
-type testResponseMap map[string]httpMethodAndResponseMap
-type httpMethodAndResponseMap map[string]parameterAndResponseMap
-type parameterAndResponseMap map[string]string
+type testResponseMap map[string][]response
+
+type response struct {
+	method   string
+	params   string
+	response string
+}
+
+type containerListInfo struct {
+	Id    string `json:"Id"`
```

Comment:
I can appreciate that but from a maintenance standpoint I think it's better to remove the duplication. I don't feel super strongly about this though.